salt/top.sls

```diff
@@ -157,6 +154,9 @@ base:
     - fluentd
     - fluentd.plugins
     - fluentd.config
+  'purpose:sandbox':
+    - match: grain
+    - edx.django_user
```

Comment:
This will need additional states defined to ensure that the ansible playbooks get deployed, Consul gets set up, etc.

Reply:
I would assume consul is getting triggered by the environment grain and does not need to be setup separately in this case, no?

Reply:
Most likely